src/data/tokenConfig.ts

```diff
@@ -100,22 +101,15 @@ export const depositAssetTokenList = [
   "RAI",
   "sUSD",
   "TUSD",
-  "USDC",
   "USDT",
 ]
 
-export const depositTokenListWithWethWbtc = [
-  ...depositAssetTokenList,
-  "WETH",
-  "WBTC",
-]
-
 export const depositAssetTokenConfig: Token[] = tokenConfig.filter(
   (token) => depositAssetTokenList.includes(token.symbol)
 )
 
 export function getTokenConfig(tokenList: string[]) {
-  return tokenConfig.filter((token) =>
-    tokenList.includes(token.symbol)
+  return tokenList.map((list) =>
+    tokenConfig.find((token) => token.symbol === list)
```

Comment:
non blocking, but this is the same logic

Reply:
Is not the same, because with this, the order is based on the passed array not from tokenConfig

Reply:
I think it's the same 
prev: filter the `tokenConfig`by "is the `token` in the `tokenList` array. 
yours: map through the `tokenList` array and return the `tokenConfig` that has the `symbol` same as `tokenList` key 
You just change how it creates the output array.

Reply:
but it's the same output so no problem then

Reply:
ah I see it now, we need the sorting and you ordered it from `tokenList` ok